shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Tick.hs

```diff
@@ -126,3 +142,45 @@ instance
   Embed (RUPD era) (TICK era)
   where
   wrapFailed = RupdFailure
+
+{------------------------------------------------------------------------------
+-- TICKF transition
+
+-- This is a variant on the TICK transition called only by the consensus layer
```

Comment:
(Those leading `--` are not needed, it might be prettier without them.)

Reply:
Ah, true. We are very inconsistent with block comments, I think I hadn't even noticed that we were in one.